src/blog/lagom-1-3-7.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Lagom 1.3.7 is released!
+date: 2017-08-18
+author_github: rcavalcanti
+tags: news
+summary: >
+    The Lagom team has released Lagom 1.3.7
+---
+
+The Lagom team has released Lagom 1.3.7. For a list of the specific changes in Lagom 1.3.7, see the [change log](/changelog.html).
+
+As an overview, this release contains the following:
+
+- Update Akka to 2.4.20 - (Security Patch Release)[http://akka.io/blog/news/2017/08/10/akka-2.4.20-released] 
+- Changed logging default settings - file appender removed (see details below)
+- Experimental Service Gateway based on akka-http
+- Storing read-side processor offsets for unhandled events
+- Upgrade to latest sbteclipse
```

Comment:
I don't think this is worth highlighting here, since it only affects people who want to develop the Lagom code itself in Eclipse, not people developing Lagom services.